db/src/rocksdb.rs

```diff
@@ -15,7 +15,7 @@ use std::sync::Arc;
 //      - If the data can be migrated manually: update "x.y1.z" to "x.y2.0".
 //      - If the data can not be migrated: update "x1.y.z" to "x2.0.0".
 pub(crate) const VERSION_KEY: &str = "db-version";
-pub(crate) const VERSION_VALUE: &str = "0.1400.0";
+pub(crate) const VERSION_VALUE: &str = "0.1500.0";
```

Comment:
```suggestion
pub(crate) const VERSION_VALUE: &str = "0.1401.0";
```

1500 is reserved for the 0.15.0 release.

Reply:
This is a breaking change that needs SDK updates as well, I think we should leave it till the next version, correct?